server/routes/serviceProviderRoutes.ts

```diff
@@ -137,22 +145,22 @@ export default function serviceProviderRoutes(router: Router, services: Services
     serviceProviderReferralsController.showEndOfServiceReportConfirmation(req, res)
   )
   get(router, '/referrals/:id/supplier-assessment/schedule/start', (req, res) =>
-    serviceProviderReferralsController.startSupplierAssessmentAppointmentScheduling(req, res)
+    appointmentsController.startSupplierAssessmentAppointmentScheduling(req, res)
   )
   get(router, '/referrals/:id/supplier-assessment/schedule/:draftBookingId/details', (req, res) =>
-    serviceProviderReferralsController.scheduleSupplierAssessmentAppointment(req, res)
+    appointmentsController.scheduleSupplierAssessmentAppointment(req, res)
   )
   post(router, '/referrals/:id/supplier-assessment/schedule/:draftBookingId/details', (req, res) =>
-    serviceProviderReferralsController.scheduleSupplierAssessmentAppointment(req, res)
+    appointmentsController.scheduleSupplierAssessmentAppointment(req, res)
   )
   get(router, '/referrals/:id/supplier-assessment/schedule/:draftBookingId/check-answers', (req, res) =>
-    serviceProviderReferralsController.checkSupplierAssessmentAnswers(req, res)
+    appointmentsController.checkSupplierAssessmentAnswers(req, res)
   )
   post(router, '/referrals/:id/supplier-assessment/schedule/:draftBookingId/submit', (req, res) =>
-    serviceProviderReferralsController.submitSupplierAssessment(req, res)
+    appointmentsController.submitSupplierAssessmentAppointment(req, res)
   )
   get(router, '/referrals/:id/supplier-assessment', (req, res) =>
-    serviceProviderReferralsController.showSupplierAssessmentAppointment(req, res)
+    appointmentsController.showSupplierAssessmentAppointment(req, res)
   )
   get(router, '/referrals/:id/supplier-assessment/scheduled-confirmation', (req, res) =>
```

Comment:
Perhaps this should also be moved into the appointments controller as well?

Reply:
Oops, missed that one - will do that as part of this PR as it's the same journey. Thanks